### PR TITLE
chore(examples): upgrade E-commerce example

### PR DESCRIPTION
**Pull request by francoischalifour**
This upgrades the E-commerce example to InstantSearch 3.0.0.

Demo: https://deploy-preview-3374--instantsearchjs.netlify.com/examples/e-commerce/

### CODE REVIEW

File: docgen/src/examples/e-commerce/index.html [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
+stylesheets:
+  - https://unpkg.com/instantsearch.css@7.1.0/themes/algolia-min.css
 ---
+
 <div class="container-fluid">
   <header class="content-wrapper">
-    <a href="https://community.algolia.com/instantsearch.js/" class="is-logo"><img src="logo-is.png" width="40"></a>
+    <a href="https://community.algolia.com/instantsearch.js/" class="is-logo"
+      ><img src="logo-is.png" width="40"
+    /></a>
```
Comment (tkrugg) [inline docgen/src/examples/e-commerce/index.html]: shall we run prettier on this thing?

Comment (francoischalifour) [reply docgen/src/examples/e-commerce/index.html]: Prettier does that actually.

File: docgen/src/examples/e-commerce/search.js [hunk 2]
```diff
@@ -1,45 +1,69 @@
...
 
 var search = instantsearch({
   indexName: 'instant_search',
   searchClient: algoliasearch('latency', '6be0576ff61c053d5f9a3225e2a90f76'),
-  routing: true
+  routing: true,
 });
 
+const customSearchbox = instantsearch.connectors.connectSearchBox(
+  ({ refine, widgetParams }, isFirstRender) => {
```
Comment (samouss) [inline docgen/src/examples/e-commerce/search.js]: Not sure about the usage of ES2015+ for those demos. IIRC they are not transpiled.

Comment (francoischalifour) [reply docgen/src/examples/e-commerce/search.js]: You're right!

**Comment by francoischalifour** [comment]
The example was converted to ES5.

**Pull request merged**